Gruntfile.js

```diff
@@ -1,21 +1,35 @@
 module.exports = function(grunt) {
 	var libs = "client/js/libs/**/*.js";
+
 	grunt.initConfig({
 		watch: {
 			files: libs,
 			tasks: ["uglify"]
 		},
+		browserify: {
+			dist: {
+		    	files: {
+		      		'client/js/compiled/lounge.js': [libs, 'client/js/libs/*.js', 'client/js/lounge.js', 'client/js/lounge.templates.js']
```

Comment:
If you require all the libs in lounge.js, you only need to do

'client/js/compiled/lounge.js': client/js/lounge.js

Which is much neater, imo.